docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md

```diff
@@ -31,94 +46,269 @@ It applies to Intel Core Ultra and Core 12 - 14 gen integrated GPUs (iGPUs), as
   pip install dpcpp-cpp-rt==2024.0.2 mkl-dpcpp==2024.0.0 onednn==2024.0.0
   ``` -->
 
-* Download and install the [**Intel oneAPI Base Toolkit**](https://www.intel.com/content/www/us/en/developer/tools/oneapi/base-toolkit-download.html?operatingsystem=window&distributions=offline). During installation, you can continue with the default installation settings.
+Download and install the [**Intel oneAPI Base Toolkit**](https://www.intel.com/content/www/us/en/developer/tools/oneapi/base-toolkit-download.html?operatingsystem=window&distributions=offline). During installation, you can continue with the default installation settings.
 
-  > <img src="https://llm-assets.readthedocs.io/en/latest/_images/quickstart_windows_gpu_oneapi_offline_installer.png"  width=90%; />
+<img src="https://llm-assets.readthedocs.io/en/latest/_images/quickstart_windows_gpu_oneapi_offline_installer.png"  width=100%; />
 
 ## Setup Python Environment
 
-* Visit [Miniconda installation page](https://docs.anaconda.com/free/miniconda/), download the **Miniconda installer for Windows**, and follow the instructions to complete the installation.
+Visit [Miniconda installation page](https://docs.anaconda.com/free/miniconda/), download the **Miniconda installer for Windows**, and follow the instructions to complete the installation.
+
+<div align="center">
+<img src="https://llm-assets.readthedocs.io/en/latest/_images/quickstart_windows_gpu_5.png"  width=70%/>
+</div>
+
+After installation, open the **Anaconda Prompt**, create a new python environment `llm`:
+```cmd
+conda create -n llm python=3.9 libuv
+```
+Activate the newly created environment `llm`:
+```cmd
+conda activate llm
+```
+  
+## Install `bigdl-llm`
+
+With the `llm` environment active, use `pip` to install `bigdl-llm` for GPU:
+Choose either US or CN website for `extra-index-url`:
+* US: 
+   ```cmd
+   pip install --pre --upgrade bigdl-llm[xpu] --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/us/
+   ```
+* CN:
+   ```cmd
+   pip install --pre --upgrade bigdl-llm[xpu] --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/cn/
+   ```
+
+```eval_rst
+.. note::
 
-  > <img src="https://llm-assets.readthedocs.io/en/latest/_images/quickstart_windows_gpu_5.png"  width=50%; />
+  If you encounter network issues while installing IPEX, refer to `this guide <https://bigdl.readthedocs.io/en/latest/doc/LLM/Overview/install_gpu.html#install-bigdl-llm-from-wheel>`_ for troubleshooting advice.
+```
 
-* After installation, open the **Anaconda Prompt**, create a new python environment `llm`:
+You can verify if bigdl-llm is successfully by simply running a few lines of code:
```

Comment:
```
## Verify Installation
You can verify if bigdl-llm is successfully by simply running a few lines of code:
...
```

Reply:
Thank you for the comments. Have updated :)